open-craft/opencraft
# SE-3041: Prevent user from commiting changes to deploy before the first instance is ready

Description goes here. e.g. This PR is about updating Mattermost to 5.21.1.

**JIRA tickets**: https://tasks.opencraft.com/browse/SE-3041

**Testing instructions**:

1. Set up OCIM
2. Register with a new user
3. Attempt to change something as a user before the instances are set up. The option to do so should not be available with an appropriate message in the status field.

**Reviewers**
